bazel-mode.el

```diff
@@ -21,9 +21,20 @@
 ;;
 ;;; Code:
 
+(defconst bazel-mode-syntax-table
+  (let ((table (make-syntax-table)))
+	; single and multiple line comment start
+	(modify-syntax-entry ?# "< 14" table)
```

Comment:
Why `14`? Bazel has only one comment style with a single comment start character, right?

Reply:
Yeah, this was a mistaken reading of the documentation on syntax flags. Removed.